app/CommandLineParser.cs

```diff
@@ -62,6 +63,7 @@ class CommandLineParser : ICommandLineParser
         // Credits supports just the List verb.
         { Noun.Credits, new HashSet<Verb> { Verb.List } },
         { Noun.WorkItem, new HashSet<Verb> { Verb.List, Verb.Create, Verb.Edit, Verb.Delete } },
+        { Noun.Progress, new HashSet<Verb> { Verb.List, Verb.Create, Verb.Edit, Verb.Delete} },
```

Comment:
Since this issue only handles edits and delete, we should not include Create or List.